lexicon.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Introduction
+
+---
+
+# Lexicon
+
+The following is a list of terms and their definitions used in the context of this project. It will be updated as new terms are introduced. The goal is to provide a clear understanding of the terms used in the project and get everyone on the same page. Most terms are associated with SOLID (Social Linked Data).
+
+## A
+
+|Term|Abbriviation|Description|
+|---|---|---|
+|**Access Control List**|ACL|A list of permissions attached to an object that specifies which users or system processes can access that object and what operations they can perform. (Optional)|
+|**Action**||An entity (usually a person, group, or application) that can perform actions on resources.|
+|**Access Control Policy**|ACP|A policy-based access control mechanism that allows more fine-grained and flexible access control compared to ACL.|
+|**Append**||The permission to add new data to a resource.|
+
+## C
+
+|Term|Abbriviation|Description|
+|---|---|---|
+|**Constraint**||A limitation or restriction placed on a rule or action.|
+|**Container**||A type of resource in Solid that can hold other resources, similar to a directory in a file system.|
+|**Context**||The environment or setting in which a resource or action exists or occurs.|
+|**Control**||The permission to manage access to a resource, including setting permissions for other users.|
+
+## D
+
+|Term|Abbriviation|Description|
+|---|---|---|
+|**Data Pod**||Personal online datastore where users store their data in a Solid ecosystem. Each pod is controlled by the individual it belongs to and is hosted by a certain provider.|
+
+## F
+
+|Term|Abbriviation|Description|
+|---|---|---|
+|**Friend of a Friend**|FOAF|A machine-readable ontology describing persons, their activities, and their relations to other people and objects. (Ontology: a formal representation of a set of concepts within a domain and the relationships between those concepts)|
```

Comment:
Ontology should ideally be a separate item.

Reply:
Didn't spot that, will fix that